src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -197,3 +197,35 @@ def test_self_cal(session):
         session.self_cal()
     except nidmm.Error as e:
         assert False
+
+
+def test_configure_functions(session):
```

Comment:
Should these be broken in to separate functions? I'm not sure what the "Pythonic" way would be...

Reply:
separate